app/controllers/wiki_controller.rb

```diff
@@ -456,9 +456,7 @@ def update_node_attributes
         main_image.save
       end
 
-      if params[:main_image].to_i == 0
-        @node.main_image_id = nil
-      elsif params[:main_image].present? && img = Image.find(params[:main_image])
+      if params[:main_image].present? && img = Image.find(params[:main_image])
```

Comment:
Hi, looks like you deleted something here! There shouldn't be any edits beyond changing the t functions.

Reply:
> Hi, looks like you deleted something here! There shouldn't be any edits beyond changing the t functions.

yea made the required changes in another pull request, I have sent can please review it...also can I know that the changes I made are correct or not?